railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -1,14 +1,66 @@
 require "minitest"
+require "method_source"
+require "rake/file_list"
 require "rails/test_unit/reporter"
+require "rails/test_unit/test_requirer"
 
-def Minitest.plugin_rails_init(options)
-  self.reporter << Rails::TestUnitReporter.new(options[:io], options)
-  if $rails_test_runner && (method = $rails_test_runner.find_method)
-    options[:filter] = method
+module Minitest
+  def self.plugin_rails_options(opts, options)
+    opts.separator "Usage: bin/rails test [options] [file or directory]"
+
+    opts.on("-e", "--environment [ENV]",
+            "Run tests in the ENV environment") do |env|
+      options[:environment] = env.strip
+    end
+
+    opts.on("-n", "--name [FILENAME:LINE]",
+            "Run a single test by appending the line number to filename:\n\n" \
+            "bin/rails test test/models/user_test.rb:27") do |name|
```

Comment:
I'm a bit confused by this option. Isn't `-n` supposed to name the method to run? Also the example does not use the flag at all. This seems to be more an explanation of the `[file or directory]` argument at the top.